### ISSUE TITLE
add more keyboard shortcuts for "OK"

### ISSUE THREAD

**Issue by mah11** [pombase/canto#2011]
This follows directly (albeit not swiftly) on from #1236, and is tangentially related to #1931:

The keyboard shortcut `alt-enter` mimics clicking the "OK" button in two popups - the main "quick add" annotation box, and the "add allele details" box. It would make my life load easier if the rest of the "OK" buttons could also be effectively clicked via `alt-enter`.

**Comment by kimrutherford** [comment]
Sorry this has taken so long.  I've added alt-enter handling in a bunch more dialogs.  I'm sure I've missed some so please know me know as you find cases where alt-enter doesn't work but should.

**Comment by mah11** [comment]
Alt-enter does now close many more pop-ups than it used to, and that has great potential. But I think it's got a bug: When I click alt-enter to close one annotation pop-up, it does close ... BUT a bunch more pop-ups appear and have to be cancelled.

This happens for annotation "quick add" and "edit" pop-ups, for every annotation type I tried (so far I've checked most but not all). On the summary page 9 (yep, NINE) pop-ups need to be dismissed, one for each annotation type. On a gene-specific page it's 8, one for each type except phenotype.

Note that genotype pop-ups are unaffected. Alt-enter works as it always has for allele details, and now also works as expected on the add/edit background popup.
